src/runtime/internal/utils.ts

```diff
@@ -60,36 +60,35 @@ export function normalizeError(error: any, isDev?: boolean) {
   // TODO: investigate vercel-edge not using unenv pollyfill
   const cwd = typeof process.cwd === "function" ? process.cwd() : "/";
 
-  // Hide details of unhandled/fatal errors in production
-  const hideDetails = !isDev && (error.unhandled || error.fatal);
-
-  const stack = hideDetails
-    ? []
-    : ((error.stack as string) || "")
-        .split("\n")
-        .splice(1)
-        .filter((line) => line.includes("at "))
-        .map((line) => {
-          const text = line
-            .replace(cwd + "/", "./")
-            .replace("webpack:/", "")
-            .replace("file://", "")
-            .trim();
-          return {
-            text,
-            internal:
-              (line.includes("node_modules") && !line.includes(".cache")) ||
-              line.includes("internal") ||
-              line.includes("new Promise"),
-          };
-        });
+  const stack =
+    !isDev && (error.unhandled || error.fatal)
```

Comment:
do we want to process the stack on handled/non-fatal errors in production, even though it will not be used at all in production? it feels like it will be more performant not to process the stack at all.

```suggestion
    !isDev
```